app/decorators/html_note.rb

```diff
@@ -0,0 +1,17 @@
+class HtmlNote < ApplicationDecorator
+  delegate_all
+
+  def title
+    "<h1>#{object.title}</h1>"
+  end
+
+  def content
+    @content ||= HtmlNote.markdown_engine.render(object.content)
+  end
+
+  def self.markdown_engine
+    @markdown_engine ||= Redcarpet::Markdown.new(
```

Comment:
What do you think about creating `MarkdownService` and move this logic to this service and use like `MarkdownService.render(content)`